app/MMB/src/scenes/timeline/TimelineEventIcon.js

```diff
@@ -2,25 +2,27 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import { StyleSheet, Color, Icon } from '@kiwicom/mobile-shared';
+import { StyleSheet, Color, Icon, TextIcon } from '@kiwicom/mobile-shared';
 
 const size = 17;
 
 export type Props = {|
-  +name: string,
+  +type: 'Icon' | 'TextIcon',
```

Comment:
I think we should rather pass in the Icon component here.

Reply:
Ok, I will refactor this.